docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -257,3 +271,19 @@ The AKO supports Gateway objects with a single IPv4 address. The user can config
   ```
 
 **NOTE:** Currently, the address of type IPAddress is only supported. The length of the addresses is also limited to a single address.
+
+#### Kubernetes Service types
+
+Currently, only `ClusterIP` and `NodePort` Service types are supported.
+
+#### Container Network Interface (CNI) providers
+
+Currently, only `Calico` is the supported CNI provider.
```

Comment:
Why are we mentioning CNI thing here? Static route creation capability is provided by main AKO so should CNI matter in AKO-GW?